specs/Deployment/DeploymentAPI.md

```diff
@@ -129,6 +129,53 @@ again.
     }
 ```
 
+### DeploymentOptions
+
+#### ForceTargetApplicationShutDown
+
+When this API may install newer version of one or more of the packages (i.e. an update) and
+if there are any currently running process(es) associated with that package(s) (in other words,
+if any of the packages to be updated are currently in use), then this API will fail installing
+that package. But this update to the packages can be forced by using DeploymentOptions object
+and setting ForceTargetApplicationShutDown option before passing it to this API. This option
+when set will shutdown the application(s) associated with the package, update the package
+and restart the application(s).
+
+[ToDo] If this option is set when updating main package, then whether the out of process
```

Comment:
`[Todo]` ?

Reply:
Sorry.. it should be //TODO:

Reply:
More like OPENISSUE: If this option...handle explicitly restarting it. Under investigation.